bin/fixedcase/truelist

```diff
@@ -1738,171 +11615,1177 @@ Universal Dependency
+Yawiyo
+Yawuru
+Yaygir
+Yazd
+Yazgulyam
+YeDebub Bihēroch Bihēreseboch na Hizboch
+Yecuatla Totonac
 Yedda
+Yei
+Yekhee
+Yekora
+Yela
+Yele
+Yelmek
+Yelogu
 Yelp.com
+Yemba
+Yemen
+Yemsa
+Yendang
+Yeni
+Yeniche
+Yerakai
+Yeretuar
+Yerong
+Yerukula
+Yerushalayim Al Quds
+Yetfa
+Yevanic
+Yevlax
+Yevreyskaya avtonomnaya oblast'
+Yeyi
 Yiddish
+Yidgha
+Yidiny
 Yih
+Yil
 Yimam
+Yimas
+Yimchungru Naga
+Yinbaw Karen
+Yinchia
+Yindjibarndi
+Yindjilandji
+Yine
+Yinggarda
+Yinhawangka
+Yiningayi
+Yintale Karen
+Yinwum
+Yir Yoront
+Yirandali
+Yis
+Yitha Yitha
+Yoba
+Yobe
+Yocoboué Dida
+Yogad
 Yogatama
+Yogyakarta
+Yoidik
+Yokuts
+Yola
+Yoloxochitl Mixtec
+Yolŋu Sign Language
+Yom
+Yombe
+Yomou
+Yonaguni
+Yong
+Yongbei Zhuang
+Yongkom
+Yongnan Zhuang
 Yongning
+Yonne
+Yopno
 Yor
+Yora
 Yorem
 Yorick
+York
+Yorkshire
+Yoro
+Yoron
+Yorta Yorta
 Yoruba
+Yorùbá
+Yosondúa Mixtec
+Yotti
+Youjiang Zhuang
+Youle Jinuo
+Younuo Bunu
+Youssoufia
+Yout Wam
 Youtube
+Yoy
+Yozgat
+Yuanga
+Yucatec Maya Sign Language
+Yucateco
+Yucatán
 Yuchi
+Yucuañe Mixtec
+Yucuna
 Yue
+Yug
+Yugambal
 Yugoslavia
+Yugoslavian Sign Language
+Yugul
+Yugur
+Yuhup
+Yukaghir
+Yuki
+Yukon Territory
+Yukpa
+Yukuben
+Yulu
+Yumbe
+Yunlin
+Yunnan Sheng
 Yupik
+Yuqui
+Yuracare
+Yurats
+Yurok
+Yuru
+Yurutí
+Yutanduchi Mixtec
+Yuwana
+Yuyu
+Yvelines
+Ywom
+Yámana
+Yên Bái
+Yāpanaya
+Zaachila Zapotec
+Zabajkal'skij kraj
+Zabana
+Zacapa
+Zacatecas
+Zacatepec Chatino
+Zachodniopomorskie
+Zadarska županija
 Zadeh
 Zaghawa
+Zaghouan
+Zagora
+Zagorje ob Savi
+Zagrebačka županija
 Zaidan
+Zaiwa
+Zajas
+Zaječarski okrug
+Zakarpats'ka Oblast'
+Zakhring
+Zakynthos
+Zala
+Zalaegerszeg
+Zalingei
+Zambales
+Zambezia
+Zambia
+Zambian Sign Language
+Zamboanga Chavacano
+Zamboanga Peninsula
+Zamboanga Sibugay
+Zamboanga del Norte
+Zamboanga del Sur
+Zamfara
+Zamora
+Zan Gula
+Zanaki
+Zande
+Zangskari
+Zangwal
+Zaniza Zapotec
+Zanjān
+Zanzan
+Zapadnobački okrug
+Zapadnohercegovački kanton
+Zaporiz'ka Oblast'
+Zapotec
+Zaqatala
 Zara
+Zaragoza
+Zaramo
 Zaremba
+Zari
+Zarma
+Zarphatic
+Zauzou
+Zavrč
+Zay
+Zayein Karen
 Zaza
+Zazao
 Zealand
+Zeeland
+Zeem
+Zeeuws
+Zelenikovo
 Zeller
 Zeman
+Zemba
+Zeme Naga
+Zemgalian
+Zenag
+Zenaga
+Zenzontepec Chatino
+Zerenkel
+Zhaba
+Zhambyl oblysy
 Zhang
+Zhejiang Sheng
+Zhemgang
 Zhidao
+Zhire
+Zhoa
 Zhou
 Zhuang
+Zhytomyrs'ka Oblast'
+Zia
 Ziai
+Zialo
+Ziguinchor
+Zigula
 Zika
+Zilupes novads
+Zimakani
+Zimba
+Zimbabwe
+Zimbabwe Sign Language
+Zinder
+Zinza
 Zipf
 Zipf's
 Zipfian
 Zipporah
+Zire
+Ziriya
+Ziro
+Zizilivakan
+Zlatiborski okrug
+Zlín
+Zlínský kraj
+Znojmo
+Zo'é
+Zokhuo
+Zomba
+Zondoma
+Zonguldak
+Zoogocho Zapotec
 Zoph
+Zoroastrian Dari
+Zotung Chin
+Zou
+Zoundwéogo
+Zreče
+Zrnovci
 Zubiaga
+Zug
+Zulia
 Zulu
+Zumaya
+Zumbun
 Zuni
 Zuojiang
 Zurich
-Zürich
 Zyphe Chin
 Zyrian
+Záparo
+Zürich
+Zābul
+Zəngilan
+Zərdab
+Z̧ufār
+the Congo
+the Gambia
+the Marshall Islands
+the Niger
+the Philippines
+the Sudan
+unitatea teritorială din Stînga Nistrului
+wallonne
```

Comment:
I assume this was from "Régione wallonne" or similar? Anyway no need to include if it's lowercase.

Reply:
It appears to be just part of the ISO list of geographic subdivisions. Maybe it is listed as an alternative name. But you are right of course, it can be dropped.